src/components/CCPForm/CompleteButton.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import Button from '@material-ui/core/Button';
+import { makeStyles } from '@material-ui/core/styles';
+import { Colours } from '../../styles/Constants';
+
+const useEventButtonStyles = makeStyles({
+  root: {
+    minWidth: '228px',
+    minHeight: '48px',
+    fontSize: '18px',
+    color: Colours.White,
+    backgroundColor: Colours.Secondary,
+  },
+  disabled: {
+    cursor: 'not-allowed !important',
+    'pointer-events': 'all !important',
+  },
+});
+
+const CompleteButton: React.FC<{
```

Comment:
is it possible to pull out one of our existing buttons and make it a common component instead of creating this entirely new button?
e.g. https://github.com/uwblueprint/paramedics-react/blob/master/src/components/EventCreationPage/NextButton.tsx

Reply:
cc: @JasonYG since you made that component originally

Reply:
yeah good idea, we should probably pull these buttons out into a common component with an interface that's something like: `<CommonButton disabled={...} onClick={...} title={...} />`